src/modules/iotjs_module_tls.c

```diff
@@ -375,10 +378,21 @@ JS_FUNCTION(TlsEnd) {
   return jerry_create_undefined();
 }
 
+JS_FUNCTION(ErrName) {
+  DJS_CHECK_ARGS(1, number);
+
+  int errorcode = JS_GET_ARG(0, number);
+  char err[50];
```

Comment:
Actually we have the same function at https://github.com/Rokid/ShadowNode/blob/0c15a5f58cd8b551cef51aa4f1cd7273718e9e28/src/modules/iotjs_module_tls.c#L82, I think we need to make them work together :)

Reply:
The core of both function is `mbedtls_strerror(int, char*, size_t)`. Is it comprehensive to do another wrap over it?